app/templates/admin/createEvent.html

```diff
@@ -333,4 +352,37 @@ <h5 class="modal-title" id="warningModel">Delete Event</h5>
       </div>
     </div>
   </div>
+<!-- Modal for Renew Event -->
+  <form action="/event/{{eventData.id}}/renew" method='post'>
+
+  <div class="modal fade" id="renewWarning" tabindex="-1" aria-labelledby="renewModel" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="renewModel">Renew Event</h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+        <div class="modal-body">
+          <p> Are you sure you want to renew this event?</p>
+          <p> A renewed event will use the current events RSVP, and location information to create a new event.</p>
```

Comment:
This is a little grammatically incorrect. It should be "A renewed event will use the current event's RSVP and location information to create a new event."

Reply:
Fixed